spec/take_away_spec.rb

```diff
@@ -0,0 +1,15 @@
+
+require "take_away"
+
+describe TakeAway do
+  subject(:take_away) { described_class.new(menu: menu) }
```

Comment:
you might need to switch the order these are in -- I had that problem because I was putting a double in a double in a double and creating them in the wrong order